scripts/deploy.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+set -e
+
+./scripts/build.sh
```

Comment:
Hm, I must have missed this in our discussion offline, but why is it that `npm run build` doesn't work in this context? Because of the file extension?